app/views/user/list.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Here is a list of users</h1>
```

Comment:
- [ ] Kindly rename the folder containing this file from `user` to `users` to match the controller name.

- [ ] This file name should be `index.html.erb`. Kindly rename the file.